tests/api_tests/base/test_views.py

```diff
@@ -47,6 +53,9 @@ def test_view_classes_have_minimal_set_of_permissions_classes(self):
         view_modules = [name for _, name, _ in pkgutil.iter_modules(['api'])]
         for module in view_modules:
             for name, view in inspect.getmembers(sys.modules['api.{}.views'.format(module)], inspect.isclass):
+                if getattr(view, 'permissions_exempt', False):
```

Comment:
Don't spend a lot of time on this. It's not so important that I think it should block progress.